Formula/yugabyte.rb

```diff
@@ -1,17 +1,21 @@
 class Yugabyte < Formula
   desc "High-performance distributed SQL database Yugabyte DB"
   homepage "https://yugabyte.com"
-  url "https://downloads.yugabyte.com/yugabyte-2.0.5.2-darwin.tar.gz"
-  sha256 "8d9b939fe10ba4116c569e27b5b177e9249e448a369dc92b42f5f4a41fa5c8b6"
+  url "https://downloads.yugabyte.com/yugabyte-2.0.6.0-darwin.tar.gz"
+  sha256 "96c23560845647c3dbff4e1923f4a5caec5fb34ccb2e50fccee4c19d5671f3fc"
   depends_on :java => "1.8"
   def install
-    prefix.install Dir["*"]
-    bin.install_symlink prefix/"yugabyted"
-    bin.install_symlink prefix/"postgres/bin/ysqlsh"
+    libexec.install Dir["*"]
```

Comment:
can you explain what the libexec.install does? why did we change this from prefix.install

Reply:
first, we were installing yuagbyte in `bin` directory due to which brew was creating a symbolic link of all the binary which are present in `bin` directory of yugabyte tar.gz package. Now we have moved that installation to `libexec` directory and creating symbolic link of `yugabyted` only.

Reply:
do we need to keep the install_symlink lines from before?

Reply:
yes